syntax = "proto3";
package Telegrams;

message Request {
    enum Command {
        PRINT = 0;
        BEGIN_TRAINING = 1;
        ADD_TRAINING_DATA = 2;
        PREDICT = 3;
        PREDICTION = 4;
        TRAINING_FINISHED = 5;
    }

    string transaction_id = 1;
    Command command = 2;
    string message = 3;
    repeated float training_x = 4;
    repeated float training_y = 5;
    repeated float predict_x = 6;
    repeated float prediction_y = 7;
}